src/chip.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 16,
+    },
+}));
+
+type ChipProps = {
+    children: string;
+    icon?: React.ReactNode;
+    onClose?: () => void;
+};
+
+const Chip: React.FC<ChipProps> = ({children, icon, onClose}) => {
+    const classes = useStyles();
+    const {colors, texts} = useTheme();
+
+    return (
+        <Box
+            className={classes.container}
+            paddingY={4}
+            paddingLeft={icon ? 8 : 12}
+            paddingRight={onClose ? 8 : 12}
+        >
```

Comment:
missing style for the background of the chip? have to be `backgroundContainer`

Reply:
done